Tools/MiniBrowser/MiniBrowser.xcodeproj/project.pbxproj

```diff
@@ -19,8 +33,33 @@
 		BC329498116A941B008635D0 /* WK2BrowserWindowController.m in Sources */ = {isa = PBXBuildFile; fileRef = BC329497116A941B008635D0 /* WK2BrowserWindowController.m */; };
 		BC72B89511E57E07001EB4EA /* MainMenu.xib in Resources */ = {isa = PBXBuildFile; fileRef = 1DDD58150DA1D0A300B32029 /* MainMenu.xib */; };
 		BC72B89611E57E0F001EB4EA /* BrowserWindow.xib in Resources */ = {isa = PBXBuildFile; fileRef = BC3294A2116A9852008635D0 /* BrowserWindow.xib */; };
+		DD403C5528500F8C00D899FC /* WebKit.framework in Product Dependencies */ = {isa = PBXBuildFile; fileRef = DD403C5328500F6400D899FC /* WebKit.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
 /* End PBXBuildFile section */
 
+/* Begin PBXContainerItemProxy section */
+		DD403C8B2851B14300D899FC /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = 29B97313FDCFA39411CA2CEA /* Project object */;
+			proxyType = 1;
+			remoteGlobalIDString = 8D1107260486CEB800E47090;
+			remoteInfo = MiniBrowser;
+		};
+/* End PBXContainerItemProxy section */
+
+/* Begin PBXCopyFilesBuildPhase section */
```

Comment:
I wonder if we can just link to WebKit to establish the dependency.

Reply:
Looks like we already do link it in the correct place: https://github.com/WebKit/WebKit/blob/63ef77d40604321c37f05572e58a94061dd3605d/Tools/MiniBrowser/Configurations/MiniBrowser.xcconfig#L32

This was likely only necessary to prevent failures when unintentionally building MiniBrowser for other platforms, and the aggregate target now prevents us from doing that.